frontend/packages/ux-editor/src/components/UnsupportedVersionMessage/UnsupportedVersionMessage.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import classes from './UnsupportedVersionMessage.module.css';
+import { Alert, Heading, Paragraph } from '@digdir/design-system-react';
+import { useTranslation } from 'react-i18next';
+
+export interface UnsupportedVersionMessageProps {
+  version: string;
+  closestSupportedVersion: string;
+  category: 'too-old' | 'too-new';
+}
+
+const getBodyTextKeys = (category: 'too-old' | 'too-new') => {
```

Comment:
Kudos, love to see that you structure the data first and then render the result the same way regardless of it one or multiple texts. 👏